.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "plugin:react-hooks/recommended"
+    // "plugin:@typescript-eslint/recommended"
```

Comment:
can we delete this commented out? It makes it look like this was something we were previously using.

Reply:
Removed